### PR TITLE
adding directory structure to template readme

### PR DESCRIPTION
**Pull request by pdg744**
Adding directory structure to the readme. 

([website#270](https://github.com/risc0/website/pull/270/) removes this content from the website)

### CODE REVIEW

File: templates/rust-starter/README.md [hunk 1]
```diff
@@ -41,6 +41,28 @@ Search this template for the string `TODO`, and make the necessary changes to im
  * Reference documentation for our Rust crates is available at [docs.rs], including the [RISC Zero zkVM crate](https://docs.rs/risc0-zkvm), the [cargo risczero crate](https://docs.rs/cargo-risczero), the [RISC Zero build crate](https://docs.rs/risc0-build), and others (the full list is available at [https://github.com/risc0/risc0/blob/main/README.md]).
  * Our [main repository](https://www.github.com/risc0/risc0).
 
+## Directory Structure
+It is possible to organize the files for these components in various ways. 
+However, in the [`cargo risczero`](https://docs.rs/cargo-risczero/0.16/cargo_risczero/) starter template we use a standard directory structure for zkVM applications, and we recommend you use this structure as well.
```
Comment (tzerrell) [inline templates/rust-starter/README.md:46]: Let's take the opportunity to be a little less prescriptive (in large projects especially there may be good project-specific reasons to organize differently).
```suggestion
However, in the [`cargo risczero`](https://docs.rs/cargo-risczero/0.16/cargo_risczero/) starter template we use a standard directory structure for zkVM applications, which we think is a good starting point for your applications.
```

**Review by tzerrell — APPROVED** [review]
Good idea

**Comment by flaub** [comment]
Is this needed for 0.18?

**Comment by tzerrell** [comment]
> Is this needed for 0.18?

No -- it is intended to be paired with https://github.com/risc0/website/pull/270/ which I discussed with @pdg744 and decided I'd hold off on reviewing that one until 0.18 is out.

File: templates/rust-starter/README.md [hunk 2]
```diff
@@ -41,6 +41,28 @@ Search this template for the string `TODO`, and make the necessary changes to im
  * Reference documentation for our Rust crates is available at [docs.rs], including the [RISC Zero zkVM crate](https://docs.rs/risc0-zkvm), the [cargo risczero crate](https://docs.rs/cargo-risczero), the [RISC Zero build crate](https://docs.rs/risc0-build), and others (the full list is available at [https://github.com/risc0/risc0/blob/main/README.md]).
  * Our [main repository](https://www.github.com/risc0/risc0).
 
+## Directory Structure
+It is possible to organize the files for these components in various ways. 
+However, in the [`cargo risczero`](https://docs.rs/cargo-risczero/0.16/cargo_risczero/) starter template we use a standard directory structure for zkVM applications, which we think is a good starting point for your applications.
```
Comment (tzerrell) [inline templates/rust-starter/README.md:46]: Update link
```suggestion
However, in the [`cargo risczero`](https://docs.rs/cargo-risczero/0.18/cargo_risczero/) starter template we use a standard directory structure for zkVM applications, which we think is a good starting point for your applications.
```

**Pull request merged**